app/lib/clients/create_employees.rb

```diff
@@ -0,0 +1,54 @@
+class Clients::CreateEmployees
+  def initialize(employees, time_off)
+    @employees = employees
+    @time_off = time_off
+    @dates = create_dates
+  end
+
+  def create_dates
+    day_before_yesterday = (Time.zone.yesterday - 1).to_formatted_s(:iso8601)
+    yesterday = Time.zone.yesterday.to_formatted_s(:iso8601)
+    today = Time.zone.now.to_formatted_s(:iso8601)
+    tomorrow = Time.zone.tomorrow.to_formatted_s(:iso8601)
+    day_afer_tomorrow = (Time.zone.tomorrow + 1).to_formatted_s(:iso8601)
+    [day_before_yesterday, yesterday, today, tomorrow, day_afer_tomorrow]
+  end
+
+  def find_birthday_employees
```

Comment:
can we connect these functions to the graphql resolvers?

Reply:
How would I go about doing that?

Reply:
Something like this https://github.com/mojotech/helios2/commit/a1e79cdab283dc8afa485a65b4af1bcf255682f0?